pkg/agent/nodeportlocal/npl_agent_test.go

```diff
@@ -551,9 +596,11 @@ func TestPodAddHostPortMultiProtocol(t *testing.T) {
 
 	value, err := testData.pollForPodAnnotation(testPod.Name, true)
 	require.NoError(t, err, "Poll for annotation check failed")
-	nplData := testData.checkAnnotationValue(value, defaultPort)
-	assert.Equal(t, nplData[0].NodePort, hostPort)
-	assert.False(t, testData.portTable.RuleExists(defaultPodIP, defaultPort))
+	expectedAnn := &ExpectedNPLAnnotation{}
+	expectedAnn.addExpectedAnnotation(defaultPort, "tcp")
```

Comment:
why do you use `"tcp"` here and `protocolTCP` almost everywhere else in these tests?

same in other places